src/config/chains.ts

```diff
@@ -1,343 +1,20 @@
-  zero: '12052',
-  phoenix: '13381',
-  mtt: '16000',
-  mtttest: '16001',
-  btcix: '19845',
-  web: '24484',
-  mintme: '24734',
-  esn: '31102',
-  got: '31337',
-  fsn: '32659',
-  nrg: '39797',
-  pc: '42069',
-  arb1: '42161',
-  'arb-nova': '42170',
-  celo: '42220',
-  emerald: '42261',
-  oasis: '42262',
-  avaeth: '43110',
-  fuji: '43113',
-  avax: '43114',
-  alfa: '44787',
-  rei: '47805',
-  tnrg: '49797',
-  trei: '55556',
-  'tkm-test0': '60000',
-  'tkm-test1': '60001',
-  'tkm-test2': '60002',
-  'tkm-test103': '60103',
-  bklv: '62320',
-  ecs: '63000',
-  'ecs-testnet': '63001',
-  tkm0: '70000',
-  tkm1: '70001',
-  tkm2: '70002',
-  tkm103: '70103',
-  ckb: '71393',
-  vt: '73799',
-  firenze: '78110',
-  maticmum: '80001',
-  'base-gor': '84531',
-  usctest: '99998',
-  usc: '99999',
-  'qkc-r': '100000',
-  'qkc-s0': '100001',
-  'qkc-s1': '100002',
-  'qkc-s2': '100003',
-  'qkc-s3': '100004',
-  'qkc-s4': '100005',
-  'qkc-s5': '100006',
-  'qkc-s6': '100007',
-  'qkc-s7': '100008',
-  bro: '108801',
-  'qkc-d-r': '110000',
-  'qkc-d-s0': '110001',
-  'qkc-d-s1': '110002',
-  'qkc-d-s2': '110003',
-  'qkc-d-s3': '110004',
-  'qkc-d-s4': '110005',
-  'qkc-d-s5': '110006',
-  'qkc-d-s6': '110007',
-  'qkc-d-s7': '110008',
-  aka: '200625',
-  alaya: '201018',
-  alayadev: '201030',
-  platondev: '210309',
-  platon: '210425',
-  ats: '246529',
-  atstau: '246785',
-  sochain: '281121',
-  sparta: '333888',
-  olympus: '333999',
-  'arb-rinkeby': '421611',
-  'arb-goerli': '421613',
-  vpioneer: '666666',
-  vision: '888888',
-  elv: '955305',
-  etho: '1313114',
-  xero: '1313500',
-  kintsugi: '1337702',
-  music: '7762959',
-  sep: '11155111',
-  tpep: '13371337',
-  ilt: '18289463',
-  qki: '20181205',
-  auxi: '28945486',
-  joys: '35855456',
-  aqua: '61717561',
-  toys: '99415706',
-  gth: '192837465',
-  'neonevm-devnet': '245022926',
-  'neonevm-mainnet': '245022934',
-  'neonevm-testnet': '245022940',
-  oneledger: '311752642',
-  tgth: '356256156',
-  dgth: '486217935',
-  ipos: '1122334455',
-  aurora: '1313161554',
-  'aurora-testnet': '1313161555',
-  'aurora-betanet': '1313161556',
-  'hmy-s0': '1666600000',
-  'hmy-s1': '1666600001',
-  'hmy-s2': '1666600002',
-  'hmy-s3': '1666600003',
-  'hmy-b-s0': '1666700000',
-  'hmy-b-s1': '1666700001',
-  'hmy-b-s2': '1666700002',
-  'hmy-b-s3': '1666700003',
-  hop: '2021121117',
-  pirl: '3125659152',
-  frankenstein: '4216137055',
-  tpalm: '11297108099',
-  palm: '11297108109',
-  ntt: '197710212030',
-  'ntt-haradev': '197710212031',
-  mole: '6022140761023',
-}
+import { networks } from '@safe-global/safe-core-sdk-utils/dist/src/eip-3770/config'
+
+/**
+ * A static shortName<->chainId dictionary
+ * E.g.:
+ *
+ * {
+ *   eth: '1',
+ *   gor: '5',
+ *   ...
+ * }
+ */
+type Chains = Record<string, string>
+
+const chains: Chains = networks.reduce((result, { shortName, chainId }) => {
+  result[shortName] = chainId.toString()
+  return result
+}, {} as Chains)
```

Comment:
Nit, but if you specify the type as a generic, you need not cast.

```suggestion
const chains = networks.reduce<Chains>((result, { shortName, chainId }) => {
  result[shortName] = chainId.toString()
  return result
}, {})
```

Reply:
Neat!